### ISSUE TITLE
Fishnet 4

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#8]
Dataset: Fishnet
Data: 10%
Model: Best iNat21 with hierarchical multitask pretraining

Use a pre-trained swin-v2-base model pre-trained on iNat21 with hierarchical multitask loss with the best validation loss, fine-tune it on Fishnet with 10% of training data, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).
